src/pages/staking-hub/onboarding.tsx

```diff
@@ -36,30 +33,56 @@ function Onboarding() {
   const dispatch = useAppDispatch();
   const onboardingStep = useAppSelector((store) => store.stakingHub.onboarding.step);
 
+  function whatIsCompletedStep(page: number) {
+    switch (page) {
+    case 2:
+    case 3:
+      return 1;
+
+    default:
+      return 0;
+    }
+  }
+
+  function whatIsCurrentStep(page: number) {
+    switch (page) {
+    case 4:
+      return 1;
+
+    default:
+      return 0;
+    }
+  }
```

Comment:
this is still confusing. I think I would go for something like an object:

```
const STEPS =  {
 CREATE_SAFE: 1,
...
}
```

Reply:
Good idea, will do sth like that in next PR